test/bin/build_images.sh

```diff
@@ -674,6 +674,10 @@ previous_version_repo=$(get_rel_version_repo "${PREVIOUS_MINOR_VERSION}")
 PREVIOUS_RELEASE_VERSION=$(echo "${previous_version_repo}" | cut -d, -f1)
 PREVIOUS_RELEASE_REPO=$(echo "${previous_version_repo}" | cut -d, -f2)
 
+# For Y-2, we only care about the version. There will always be a real repository.
```

Comment:
What will happen when 4.15 is GA'd and we stop enabling rhocp-4.14?

Reply:
We still need y-1 enabled to test upgrading 1 version, right? Why would the 4.14 repo no longer be enabled in image builder?

Reply:
Because we have `LATEST_RHOCP_MINOR="$("${SCRIPTDIR}/../../scripts/get-latest-rhocp-repo.sh")"` - i.e. single value and we template `rhocp.toml` once, so we'll only get the most recent one

Reply:
I retested the metal job and it failed:
```
sudo composer-cli compose start-ostree --parent rhel-9.2 --url http://10.192.10.141:8080/repo --ref rhel-9.2-microshift-4.14 rhel-9.2-microshift-4.14 edge-commit
ERROR: DepsolveError: DNF error occurred: MarkingErrors: Error occurred when marking packages for installation: Problems in request:
missing packages: microshift-4.14.0~rc.7-202310241737.p0.g1586504.assembly.rc.7.el9*, microshift-greenboot-4.14.0~rc.7-202310241737.p0.g1586504.assembly.rc.7.el9*, microshift-networking-4.14.0~rc.7-202310241737.p0.g1586504.assembly.rc.7.el9*, microshift-selinux-4.14.0~rc.7-202310241737.p0.g1586504.assembly.rc.7.el9*
```

today 4.15 was GA'd and it doesn't add rhocp-4.14 anymore:
```
+ 08:42:13.258183698 ./bin/build_images.sh:99 	sudo composer-cli sources info rhocp-4.15
check_gpg = true
check_repogpg = false
check_ssl = true
id = "rhocp-4.15"
name = "Red Hat OpenShift Container Platform 4.15 for RHEL 9"
rhsm = true
system = false
type = "yum-baseurl"
url = "https://cdn.redhat.com/content/dist/layered/rhel9/x86_64/rhocp/4.15/os"
```

Reply:
OK, I'll wait until #3079 merges and rebase on top of that.